Backend/Lyzer/Clients/JolpicaClient.cs

```diff
@@ -46,5 +46,32 @@ public async Task<DriverStandingsDTO> GetCurrentDriverStandings()
 
             return driverStandings;
         }
+
+        public async Task<ResultsDTO> GetLatestRaceResult()
```

Comment:
Let's also maybe change this to be GetRaceResult with a string year as a parameter. Reduces a need to update it later on when we are wanting to get the race results for a particular year

Reply:
So in contrast to my point on the controller.
I am happy to add this parameter, just for future proofing in case we did need it.

So I will add that and adjust to just use current or whatever :)

Reply:
Please look at [this](https://github.com/LittleClumsy/Lyzer/pull/41#discussion_r1875500581)